### PR TITLE
Added ability to register model schemas for a run

### LINKED ISSUE #360: [Deployment] Add ability to RegisterSchema
**Is your feature request related to a problem? Please describe.**
We currently have a SchemaGenerator but no way to store it.

**Describe the solution you'd like**
At the time at which someone would like to deploy a model to a container, they most likely don't have access to the input and output types. We need to store the types in a new table associated with the run.

I'm thinking a table called `ModelSchema` with columns:
- `ModelSchemaId`
- `RunId`
- `Name` (string, either ModelInput or ModelOutput)
- `Schema` (string)

I'm thinking a method called `RegisterSchema<TInput, TOutput>(Guid runId)`

### PR DESCRIPTION
**Pull request by aslotte**
## Resolves
Fixes #360 

## Description
Added the ability to register a schema for a model and run

### CODE REVIEW

**Comment by aslotte** [comment]
@AnoojNair - sorry for jumping ahead, I had a stab at it based on the great work you had already done in your PR. I was hoping to work on #370 today and I needed this functionality to do so, so I went ahead and added the changes to your existing PR, created a new branch and a PR for it (as I don't have write access to your branch). Hope that's okay!

File: src/MLOps.NET/Entities/Impl/ModelSchema.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        /// </summary>
+        public Guid ModelSchemaId { get; set; }
+
+        /// <summary>
+        /// RunId
+        /// </summary>
+        public Guid RunId { get; set; }
+
+        /// <summary>
+        /// Name if model schema (ModelInput or ModelOutput)
```
Comment (aslotte) [inline src/MLOps.NET/Entities/Impl/ModelSchema.cs:21]:
```suggestion
        /// Name of model schema (ModelInput or ModelOutput)
```

**Review by Brett-Parker — APPROVED** [review]
Good work

**Comment by AnoojNair** [comment]
@aslotte That's great. Thanks for picking it up.sorry, it took me a while on that feature..

**Comment by aslotte** [comment]
No worries @AnoojNair, thanks for your contributions!

**Pull request merged**